web/assets/stylesheets/application.css

```diff
@@ -370,6 +370,15 @@ img.smallogo {
 .stat p {
   font-size: 0.9em;
 }
+
+.metric {
```

Comment:
Is `fn` (formatted number) or `num` a better term since this class is used on non-metric pages?

Reply:
Sounds good!